docs/getting-started/handling-data.mdx

```diff
@@ -65,39 +65,41 @@ my_first_data_variable = DataVariable(data, cache=CacheType.GLOBAL)
 config.add_page('data page', plot_data(my_first_data_variable))
 ```
 
-- in action `extras`
+- in an `action`
 
 ```python
 import pandas
-from dara.core import DataVariable, CacheType, UpdateVariable, Variable
+from dara.core import DataVariable, CacheType, Variable, action
 
 data = pandas.DataFrame(data={'a': [1, 2, 3], 'b': [4, 5, 6]})
 my_first_data_variable = DataVariable(data, cache=CacheType.GLOBAL)
 
 target = Variable()
 
-UpdateVariable(
-    my_resolver_function,
-    variable=target,
-    extras=[my_first_data_variable]
-)
+@action
+async def my_resolver_function(ctx: action.Ctx, my_data_var):
+    # my_data_var here takes the value of my_first_data_variable
+    ...
+    ctx.update(variable=target, value=my_data_var)
```

Comment:
missing `await`

Reply:
I knew there had to be one missing somewhere, fixed!